buildall.sh

```diff
@@ -32,18 +37,26 @@ function build_images {
 	localrepo=$2
 	tmpdir=$(mktemp --tmpdir -d tmp-buildall.XXX)
 
-	echo "Building in $tmpdir"
+	echo "Building REPO=$REPO in $tmpdir"
 
 	for i in $1; do
 		DISTRO_TO_BUILD=$(basename $i)
 		OUTPUTDIR=$tmpdir/$DISTRO_TO_BUILD
 		mkdir $OUTPUTDIR
 
 		echo "Building $DISTRO_TO_BUILD"
-		TMPDIR=$OUTPUTDIR REPO=$REPO DISTRO_TO_BUILD=$DISTRO_TO_BUILD \
+
+		if [ $SERIAL_BUILD != "0" ]; then
+                    TMPDIR=$OUTPUTDIR REPO=$REPO DISTRO_TO_BUILD=$DISTRO_TO_BUILD \
+			bash -c "$build_cont . >& \
+			$OUTPUTDIR/build.log || \
+			echo \"$DISTRO_TO_BUILD build failed\""
```

Comment:
Did you test this with ctrl-c? Since  PIDS never gets set, does the SIGINT actually get passed to docker build and it stops?